core/src/main/java/de/danoeh/antennapod/core/storage/DatabaseExporter.java

```diff
@@ -53,7 +54,16 @@ public static void exportToStream(FileOutputStream outFileStream, Context contex
             if (currentDB.exists()) {
                 src = new FileInputStream(currentDB).getChannel();
                 dst = outFileStream.getChannel();
-                dst.transferFrom(src, 0, src.size());
+                long srcSize = src.size();
+                dst.transferFrom(src, 0, srcSize);
+
+                long newDstSize = dst.size();
+                if (newDstSize < srcSize) {
```

Comment:
Just noticed something else. Is there a specific reason for checking `<` instead of making sure that both files have the exactly same size?

Reply:
@jmdouglas ping :) Are you still interested in this PR?

Reply:
@ByteHamster Sorry about that